perfzero/lib/cloud_manager.py

```diff
@@ -133,9 +134,37 @@ def create(username, project, zone, machine_type, accelerator_count,
   logging.info('Successfully created gcloud computing instance %s with %s accelerator.\n',  # pylint: disable=line-too-long
                instance_name, accelerator_count)
 
+  # Wait until we can ssh to the newly created computing instance
+  cmd = 'gcloud compute ssh {} --project={} --zone={} --strict-host-key-checking=no --command="ls"'.format(instance_name, project, zone)  # pylint: disable=line-too-long
+  ssh_remaining_retries = 12
+  ssh_error = None
+  while ssh_remaining_retries > 0:
+    ssh_remaining_retries -= 1
+    try:
+      run_command(cmd, is_from_user=False)
+      ssh_error = None
+    except Exception as error:  # pylint: disable=W0703
```

Comment:
Seems that you only wants to catch the timeout error here. Do you actually want a broad except? (If you actually do, maybe disable `broad-except` instead of `W0703`)

Reply:
`run_command()` executes arbitrary command in subprocess and get an error_code in response. It seems that every command has its own error_code interpretation and there is not standard for converting between exception and error_code. Thus currently `run_command(...)` simply throws `Exception` without information of whether it is timeout error. And we will catch `Exception` here.

Yeah it is more readable to use `broad-except` instead of `W0703`. I will make the change here. Thanks!